### PR TITLE
Add Jenkinsfile

### PR DESCRIPTION
**Pull request by mjemnawaz**

### CODE REVIEW

**Comment by lewismc** [comment]
Hi @mjemnawaz I can read the code but can please explain which task you want to execute as part of the CI?
Also has this just to be run nightly or do you have some other intentions? 
thanks

**Comment by mjemnawaz** [comment]
@lewismc Right now I'm running all 3 stages that are uncommented in the Jenkinsfile, and I'm using a webhook so that it runs everytime someone pushes to GitHub, plus it also runs every 24 hours.

**Comment by lewismc** [comment]
OK great @mjemnawaz I'm trying to set this up right now. Thank you

File: Jenkinsfile [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.	
+ 
+*/
+
+pipeline {
+    agent any
+    environment {
+        registry = 'fireantci/fireant'
```
Comment (lewismc) [inline Jenkinsfile:22]: Please update

File: Jenkinsfile [hunk 2]
```diff
@@ -0,0 +1,65 @@
+/*
```
Comment (lewismc) [inline Jenkinsfile:1]: Actually @mjemnawaz please provide a README for this entire PR. Thank you

**Review by lewismc — CHANGES_REQUESTED** [review]
Good work @mjemnawaz

**Pull request merged**